src/docs/asciidoc/index.adoc

```diff
@@ -37,28 +37,14 @@ include::{samples-path}/primer/groovy/build.gradle[]
 
 Use compile-safe accessor in your `settings.gradle(.kts)` (since 1.1):
 
-```kotlin
-plugins {
-    id("build-parameters")
-}
-
-dependencyResolutionManagement {
-    repositories {
-        maven {
-            url = uri("https://repo.my-company.com")
-            credentials {
-                // username has a default and is therefore of type String
-                username = the<BuildParametersExtension>().deployment.username
-                // [Groovy DSL] buildParameters.deployment.username
-
-                // password does not have a default and is therefore of type Provider<String>
-                password = the<BuildParametersExtension>().deployment.password.get()
-                // [Groovy DSL] buildParameters.deployment.password.get()
-            }
-        }
-    }
-}
-```
+[source.multi-language-sample,kotlin]
+----
+include::{samples-path}/primer/kotlin/settings.gradle.kts[tags=build-parameters]
+----
+[source.multi-language-sample,groovy]
+----
+include::{samples-path}/primer/groovy/settings.gradle[tags=build-parameters]
+----
```

Comment:
Should we add here that building settings plugins from included builds does only work when the build is included in `pluginManagement`? I think this is an inportant aspect because I would not advice putting all build logic into an included build that's included that way. The reason being the problems with build cache configuration.